frontend/src/views/WebinairePage.vue

```diff
@@ -13,10 +13,22 @@
         :id="`video-${mainVideo.id}`"
       >
         <source :src="mainVideo.video" />
+        <track v-if="mainVideo.subtitles" label="Français" kind="subtitles" srclang="fr" :src="mainVideo.subtitles" />
         Votre navigateur ne peut pas afficher des vidéos.
       </video>
     </div>
     <p class="mt-2 mb-4" v-if="mainVideo && mainVideo.description">{{ mainVideo.description }}</p>
+    <v-alert v-if="!mainVideo.subtitles" type="error" outlined>
+      <p>
+        Cette vidéo n'a pas de sous-titres. Si vous en avez besoin, contactez-nous avec notre
+        <router-link :to="{ name: 'ContactPage' }">formulaire de contact</router-link>
+        pour qu'on puisse prioriser le sous-titrage de ce contenu.
```

Comment:
Peut-être un peu plus concis ?
```suggestion
        pour prioriser le sous-titrage de ce contenu.
```